utils/sn_flux_est.py

```diff
@@ -804,16 +938,23 @@ def est_pulsar_sn(pulsar, obsid,\
                     ERROR=logging.ERROR)
 
     parser = argparse.ArgumentParser(description="""A utility file for estimating the S/N of a pulsar in an obsid""")
+
     parser.add_argument("-o", "--obsid", type=int, help="The Observation ID (e.g. 1221399680)")
-    parser.add_argument("-p", "--pulsar", type=str, help="The pulsar's name (e.g. J2241-5236)")
+    parser.add_argument("-p", "--pulsar", type=str, nargs='+', help="The pulsar's name (e.g. J2241-5236).\
+                        Takes a single argument for flux estimation or multiple for ATNF plotting")
     parser.add_argument("-L", "--loglvl", type=str, default="INFO", help="Logger verbostity level. Default: INFO")
     parser.add_argument("-b", "--beg", type=int, default=None, help="The beginning time of observation.\
                         If None, will use beginning given by a metadata call. Default: None")
     parser.add_argument("-e", "--end", type=int, default=None, help="The end time of observation.\
                         If None, will use the end given by a metadata call. Default: None")
     parser.add_argument("--pointing", type=str, default=None, help="The pointing of the target in the format '12:34:56_98:76:54'.\
                         If None, will obtain from a call to ATNF. Default: None")
+    parser.add_argument("min_z_power", type=float, default=0.3, help="The minimum zenith normalised power used to determine if the pulsar\
+                        is in the beam or not")
     parser.add_argument("--plot_est", action="store_true", help="Use this tag to create a plot of flux estimation.")
+    parser.add_argument("--mode", type=str, help="""MODES: 'SNFE' = Estimate S/N and flux for a single pulsar in an obsid\n
+
+                                                'ATNF' = Plot the SED for any number of pulsars using data from ATNF.""")
```

Comment:
Make this clearer by expanding SED and explaining that it is not doing any flux calculations

Reply:
Why would the user think that it's doing any flux calculations in this mode?